lib/opal/cli_runners/nodejs.rb

```diff
@@ -1,66 +1,29 @@
 # frozen_string_literal: true
 require 'opal/paths'
-require 'shellwords'
 
 module Opal
   module CliRunners
-    class Nodejs
+    class Nodejs < Cmd
       NODE_PATH = File.expand_path('../stdlib/nodejs/node_modules', ::Opal.gem_dir)
 
       def initialize(options)
-        @output = options.fetch(:output, $stdout)
+        super(options, 'nodejs', {'NODE_PATH' => node_modules}, 'node')
       end
-      attr_reader :output, :exit_status
 
       def puts(*args)
         output.puts(*args)
       end
 
       def node_modules
-        paths = ENV['NODE_PATH'].to_s.split(':')
-        paths << NODE_PATH unless paths.include? NODE_PATH
-        paths.join(':')
+        NODE_PATH
```

Comment:
Previous version included `ENV['NODE_PATH']`. Even we nobody passes it it still doesn't make sense to have a method that returns a constant.

Reply:
Oops, I missed my rebase 😞